launcher/modManager/chroniclesextractor.cpp

```diff
@@ -228,29 +212,46 @@ void ChroniclesExtractor::extractFiles(int no) const
 
 void ChroniclesExtractor::installChronicles(QStringList exe)
 {
+	logGlobal->info("Installing Chronicles");
+
 	extractionFile = -1;
 	fileCount = exe.size();
 	for(QString f : exe)
 	{
 		extractionFile++;
 
+		logGlobal->info("Creating temporary directory");
 		if(!createTempDir())
 			continue;
 		
+		logGlobal->info("Copying offline installer");
+		// FIXME: this is required at the moment for Android (and possibly iOS)
+		// Incoming file names are in content URI form, e.g. content://media/internal/chronicles.exe
+		// Qt can handle those like it does regular files
+		// however, innoextract fails to open such files
```

Comment:
Iirc there's a way to get path on disk that's compatible with standard file read functions

Reply:
Possible, but haven't found it. Found one reference to Qt 6.X (6.6 I think?), but we're at Qt 5.

In any case this code was before, and it worked on all tested OS's. Added comment mostly to explain why this strange copy can't be simply removed.

Reply:
Oh, then it's probably possible only via jumping into android sdk